README.md

```diff
@@ -200,6 +200,63 @@ telescope transpile --config .telescope.json --config .telescope-ext.json
 }
 ```
 
+### Download with CLI
+
+You can also use our CLI for download. To download the proto files, `download` command.
+
+```sh
+telescope download
+```
+
+You should now see some repos cloned in `./git-modules` and proto files generated in `./protos`. These are the proto files downloaded according to your config.
+
+Examples:
+
+```sh
+# Telescope will do the download according to .json file of --config
+# Telescope will put proto into location specified by --out 
+telescope download --config ./protod.config.json --out ./git-modules
+```
+
+```sh
+# Telescope download from target repo according to --git-repo
+# in format of (i.e. <owner>/<repository> or <owner>/<repository>/<branch>)
+# Also --targets is required to specify the targets to download 
+# in format like cosmos/auth/v1beta1/auth.proto
+telescope download --git-repo target-repo --targets target-proto
+```
+
+```sh
+# ssh arg is optional, default is false
+telescope download --config ./protod.config.json --out ./git-modules --ssh true
+```
+
+
+```js
+//.protod.config.json example
+// `repos` are the repository it's going to clone
```

Comment:
there're three parts: owner, repo and branch. branch can be empty and it will use main as default.